examples/module/src/gizmo/my_gizmo.py

```diff
@@ -28,6 +28,10 @@ def new(cls, config: ComponentConfig, dependencies: Mapping[ResourceName, Resour
         gizmo.my_arg = config.attributes.fields["arg1"].string_value
         return gizmo
 
+    @classmethod
+    def validate_config(cls, config: ComponentConfig) -> List[str]:
+        return [config.attributes.fields["arg1"].string_value]
```

Comment:
Returning the value of `arg1` here will mean that the gizmo will depend on some resource `arg1` which doesn't exist. Let's talk about what we want to do in this function offline.